.docker/qgis.dockerfile

```diff
@@ -82,3 +82,7 @@ WORKDIR /
 
 # Run supervisor
 CMD ["/usr/bin/supervisord", "-c", "/etc/supervisor/supervisord.conf"]
+
+# Install python modules required to load automatically loaded
+# python plugins
+RUN pip3 install jinja2 pygments
```

Comment:
up to now other Python dependencies were installed in the dependency image.
Are you suggesting to split the dependencies used for building than the ones used at run-time?
if so, can you add a TODO in the deps images?

Reply:
I didn't think about it at all, but as the "dependencies image" is indeed called "build-deps" and documented to contain all dependencies required to build, it doesn't make sense to me to add there anything which is NOT needed for building qgis.

I'm not familiar with the deps dockerfile (I see there are 2 of them?) nor I know which of those deps are runtime rather than build-time deps. How about just filing an issue, if you know which deps are in the wrong place, to  move them ?

Reply:
OK fair enough.
Just the RUN command must be be placed before the CMD command.